sdk/communication/azure-communication-chat/tests/test_chat_client.py

```diff
@@ -27,10 +28,18 @@
 
 
 class TestChatClient(unittest.TestCase):
+    @classmethod
```

Comment:
This shouldn't be a classmethod - just move this as a separate function outside the class.

Reply:
makes sense. have moved it out